auth/services/selfsigned/validator.go

```diff
@@ -135,6 +139,22 @@ func (v validator) verifyVP(vp vc.VerifiablePresentation, validAt *time.Time) (c
 		return
 	}
 
+	// #2428: NutsEmployeeCredential trust is derived from the fact that the issuer has a trusted NutsOrganizationCredential
+	searchTerms := []vcr.SearchTerm{
+		{IRIPath: jsonld.CredentialSubjectPath, Value: vc.Issuer.String()},
+		{IRIPath: jsonld.OrganizationNamePath, Type: vcr.NotNil},
+		{IRIPath: jsonld.OrganizationCityPath, Type: vcr.NotNil},
+	}
+	nutsOrgCreds, err := v.vcr.Search(context.Background(), searchTerms, false, validAt)
+	if err != nil {
+		resultErr = fmt.Errorf("unable to check NutsEmployeeCredential trust status using NutsOrganizationCredential: %w", err)
+		return
+	}
+	if len(nutsOrgCreds) == 0 {
+		resultErr = newVerificationError("invalid NutsEmployeeCredential, issuer does not have a trusted NutsOrganizationCredential")
```

Comment:
untrusted != invalid

Reply:
I didn't want to have another "trust" in this sentence, making people think they still have to explicitly trust the NutsEmployeeCredential. Ideas?

Reply:
just drop the "invalid" at the start? (and comma)

Reply:
or `NutsEmployeeCredential rejected, ...`